torch/csrc/jit/codegen/cuda/expr_evaluator.cpp

```diff
@@ -26,7 +26,6 @@ void ExpressionEvaluator::bind(Val* value, Int::ScalarType concrete_value) {
 }
 
 c10::optional<Int::ScalarType> ExpressionEvaluator::evaluate(Val* value) {
-  FUSER_PERF_SCOPE("ExpressionEvaluator::evaluate");
```

Comment:
How much difference did you with this change?

Reply:
The difference without profiler on is quite negligible but with the profiler on the overhead of this scope tends to be quite inflated because it's called too many times. 

Also this function will be very optimized after #1086, i.e. <100ns range, which would be comparable to the perf scope overhead. So I guess it'd not be worthwhile profiling this function.